reactive/openjdk.sh

```diff
@@ -32,6 +32,17 @@ function install() {
     local java_type=$(config-get 'java-type')
     local java_major=$(config-get 'java-major')
 
+    if grep -q trusty /etc/lsb-release; then
+        # ensure that Java 8 can be installed on trusty
+        add-apt-repository -y ppa:openjdk-r/ppa
+        apt-get update -qq
+    fi
+
+    if [[ `uname -p` == 'ppc64'* ]]; then
```

Comment:
It's identical to [java-installer.sh](https://bazaar.launchpad.net/~bigdata-dev/bigdata-data/trunk/view/head:/common/noarch/java-installer.sh#L40) but if you want further proof:

```bash
$ test_cpu_arch="ppc64le"
$ if [[ "$test_cpu_arch" == 'ppc64'* ]]; then
>   echo "Power match"
> fi
Power match
```